lib/SIL/Utils/Projection.cpp

```diff
@@ -549,13 +549,21 @@ ProjectionPath::removePrefix(const ProjectionPath &Path,
 }
 
 void Projection::print(raw_ostream &os, SILType baseType) const {
-  if (isNominalKind()) {
+  if (getKind() == ProjectionKind::Struct ||
```

Comment:
Could you make this an exhaustive switch?

Reply:
Done